src/Package.js

```diff
@@ -66,33 +66,61 @@ export default class Package {
     return allDependencies;
   }
 
-  async updateDependencyVersionRange(
+  async setDependencyVersionRange(
     depName: string,
     depType: string,
-    versionRange: string
+    versionRange: string | null
   ) {
-    let prevVersionRange =
-      this.config[depType] && this.config[depType][depName];
+    let prevDeps = this.config[depType];
+    let prevVersionRange = (prevDeps && prevDeps[depName]) || null;
     if (prevVersionRange === versionRange) return;
 
+    await this.updateDependencies(depType, {
+      ...prevDeps,
+      [depName]: versionRange
+    });
+
+    let pkgName = this.config.name;
+
+    if (versionRange === null) {
+      logger.info(messages.removedPackageDependency(pkgName, depName));
+    } else if (prevVersionRange === null) {
+      logger.info(
+        messages.addedPackageDependency(pkgName, depName, versionRange)
+      );
+    } else {
+      logger.info(
+        messages.updatedPackageDependency(
+          pkgName,
+          depName,
+          versionRange,
+          prevVersionRange
+        )
+      );
+    }
+  }
+
+  async updateDependencies(
```

Comment:
Personally I would either only expose `setDependencyVersionRange` *or* `updateDependencies`, not both.
There are cases where each would work.

I think you're right in that you'll quite often be editing one dep at a time, so I'm leaning towards setDependencyVersionRange. 

Happy either way though.